ClubKey/ClubKey.Data/Entities/Models/City.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace ClubKey.Data.Entities.Models
+{
+    public class City
+    {
+        public int Id { get; set; }
+
+        [Required(ErrorMessage = "Name is required"),
+         StringLength(25, MinimumLength = 3,
```

Comment:
I think the minimum part doesn't make sense since there are cities with less than three letters in their name